repository/src/main/java/org/alfresco/service/cmr/tagging/TaggingService.java

```diff
@@ -94,6 +95,18 @@ default PagingResults<Pair<NodeRef, String>> getTags(StoreRef storeRef, PagingRe
     {
         return new EmptyPagingResults<>();
     }
+
+    /**
+     * Get a map of tags filtered by name and sorted by tag name or count

```

Comment:
Minor one - can be fixed in another iteration - javadoc here should explain what key and values are (either in general method description on in `@return` part.